style.css

```diff
@@ -11,24 +11,86 @@ body {
   font-family: sans-serif;
 }
 
-.header-page {
+header {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   padding: 24px;
 }
 
-.Hamberger {
+.hamberger {
   width: 16px;
-  top: 17px;
+  top: 12px;
   left: 4px;
-  display: flex;
   margin: 2px;
+  display: block;
   cursor: pointer;
   color: #6070ff;
   font-size: 1.7rem;
 }
 
+.Closebtn {
+  height: 12px;
+  width: 12px;
+  left: 6px;
+  z-index: 100;
+  top: 6px;
+  display: block;
+  border-radius: 0;
+  font-size: 2rem;
+  cursor: pointer;
+  position: absolute;
+  color: white;
+  margin-left: 410px;
```

Comment:
- [ ] Using `margin-left: 320px;` could be better to show the close button in the correct position, however after closing the mobile menu the close button should disappear. Please fix this issues for a better UX.

![image](https://user-images.githubusercontent.com/58642949/231283459-67b38569-0298-4acc-96a4-7b490f2abdd7.png)

![image](https://user-images.githubusercontent.com/58642949/231283487-92065532-95d0-47c2-ab6a-ff20d0def488.png)